### PR TITLE
add test file

### PR DESCRIPTION
**Pull request by alanlima**

### CODE REVIEW

File: simple_tests.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    --protocol tcp
+
+aws autoscaling create-launch-configuration \
+    --launch-configuration-name devops-init-instance-data \
+    --user-data file://user-data.txt \
+    --image-id ami-088ff0e3bde7b3fdf \
+    --instance-type t2.micro \
+    --security-groups sg-098dda5e94f8a3855
+
+aws autoscaling create-auto-scaling-group \
```
Comment (alanlima) [inline simple_tests.sh:20]: Some comment here ;)

File: simple_tests.sh [hunk 2]
```diff
@@ -0,0 +1,36 @@
+aws ec2 create-security-group \
+        --group-name c03-aws01-sg \
+        --description "Allow connection to port 80" \
+        --vpc-id vpc-027019faaf6756407 \
+        --tag-specifications "ResourceType=security-group,Tags=[{Key=Name,Value=c03-aws01-sg},{Key=DevopsClass,Value=c03}]"
+
+aws ec2 authorize-security-group-ingress \
```
Comment (alanlima) [inline simple_tests.sh:7]: another **comment here**

**Review by alanlima — COMMENTED** [review]
Hey buddy, could you please fix those items 😁

File: simple_tests.sh (hunk 2 above)
Comment (km-developer) [inline simple_tests.sh:7]: Sounds good

**Review by km-developer — APPROVED** [review]

File: simple_tests.sh [hunk 3]
```diff
@@ -0,0 +1,36 @@
+aws ec2 create-security-group \
+        --group-name c03-aws01-sg \
+        --description "Allow connection to port 80" \
+        --vpc-id vpc-027019faaf6756407 \
```
Comment (km-developer) [inline simple_tests.sh:4]: nice touch

**Pull request merged**